plugins/cilium-cni/chaining/generic-veth/generic-veth.go

```diff
@@ -216,7 +216,7 @@ func (f *GenericVethChainer) Add(ctx context.Context, pluginCtx chainingapi.Plug
 		logfields.ContainerInterface: ep.ContainerInterfaceName,
 	})
 
-	err = cli.EndpointCreate(ep)
+	_, err = cli.EndpointCreate(ep)
```

Comment:
Shouldn't Cilium always control the MAC address of the devices if this feature is enabled? I think it would be confusing if this doesn't work in generic-veth mode as well.